src/commands/batch.py

```diff
@@ -101,9 +101,9 @@ def view(batch_id, node):
         display_table([], table_data)
 
     @batch.group(help='Run a command on node(s) or group(s)')
-    @click.option('--node', '-n', metavar='NODE',
+    @click.option('--node', '-n', multiple=True, metavar='NODE',
```

Comment:
Now that multiple nodes and groups are allowed, the flags should be changed to `--nodes` and `--groups`. Possible look at having option aliases for `--node` and `--group`.

Reply:
Kk, done

Reply:
Follow up discussions, this doesn't look to be easily possible. The `--node` and `--group` flags are fine for now.